src/compiler/builder.ts

```diff
@@ -93,12 +93,14 @@ namespace ts {
         signature: string;
     }
 
-    export function createBuilder(
-        getCanonicalFileName: (fileName: string) => string,
-        getEmitOutput: (program: Program, sourceFile: SourceFile, emitOnlyDtsFiles: boolean, isDetailed: boolean) => EmitOutput | EmitOutputDetailed,
-        computeHash: (data: string) => string,
-        shouldEmitFile: (sourceFile: SourceFile) => boolean
-    ): Builder {
+    interface BuilderOptions {

```

Comment:
The function which uses this is exported, but the interface describing its arguments is not. This will be an issue when generating declarations if/when the `@internal` tag is removed from this namespace. It's probably best to export the interface in anticipation of that.